docs/migration-guides/qiskit-runtime.mdx

```diff
@@ -53,14 +71,25 @@ See these topics for instructions:
 -  [Update code that calculates expectation values](qiskit-runtime-examples#estimator-algorithm)
 -  [Common use cases (basic, parameterized, and dynamic circuits)](qiskit-runtime-use-case)
 
-
-### Step 3a: replace any `backend.run` options with `qiskit_ibm_runtime` options.
+### Step 3a: Replace any `backend.run` options with `qiskit_ibm_runtime` options
 
 See the following topics for instructions:
 
 - [Migrate options](qiskit-runtime-options)
 - Common use cases [Options section](qiskit-runtime-use-case#migrate-options)
 
+### Step 3b: If you want to access a 3rd-party provider that only exposes `backend.run`, you can still follow a primitive-compatible your workflow using the Qiskit backend primitives
```

Comment:
I think maybe a word is missing (or is there an extra word?) in this header?

Reply:
+1 on this. How about

```
Step 3: Replace the call to backend.run with a call to primitives

See these topics for instructions if you are using `qiskit-ibm-runtime`:

- Update code that performs circuit sampling
- Update code that calculates expectation values
- Common use cases (basic, parameterized, and dynamic circuits)
- Replace any backend.run options with Qiskit Runtime primitive options
    - Migrate options
    - Common use cases Options section

See the following topic for instructions if you are using a 3rd-party provider that only exposes backend.run:

...
```